search/indexing_api.py

```diff
@@ -298,15 +364,69 @@ def program_enrolled_user_mapping():
     return mapping
 
 
+def public_program_enrolled_user_mapping():
+    """
+    Builds the raw mapping data for the program-enrolled user doc type
+    """
+    mapping = Mapping(PUBLIC_USER_DOC_TYPE)
+    mapping.field("id", "long")
+    mapping.field("user_id", "long")
+    mapping.field("profile", "object", properties={
```

Comment:
It wasn't on the list of filters on the original ticket, we can always add it later if we want.